scipy/misc/_common.py

```diff
@@ -178,10 +178,11 @@ def ascent():
     """
     import pickle
     import os
+    import numpy as np
     fname = os.path.join(os.path.dirname(__file__),'ascent.dat')
     with open(fname, 'rb') as f:
         ascent = array(pickle.load(f))
```

Comment:
Since you are working on this file, I would suggest properly using NumPy with moving the import at the top and change usages.

Also here I would just pass the type directly.
```suggestion
        ascent = np.array(pickle.load(f), dtype=np.uint8)
```

Reply:
Thanks for the suggestion @tupui . Looks much more elegant than my original fix.